Configuration/Generator/python/BuToJPsiPrimePhiToJPsiPiKK_14TeV_TuneCP5_pythia8_cfi.py

```diff
@@ -0,0 +1,122 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from GeneratorInterface.EvtGenInterface.EvtGenSetting_cff import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         comEnergy = cms.double(14000.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         maxEventsToPrint = cms.untracked.int32(0),
+                         pythiaPylistVerbosity = cms.untracked.int32(0),
+                         ExternalDecays = cms.PSet(
+        EvtGen130 = cms.untracked.PSet(
+            decay_table = cms.string('GeneratorInterface/EvtGenInterface/data/DECAY_2014_NOLONGLIFE.DEC'),
+            particle_property_file = cms.FileInPath('GeneratorInterface/EvtGenInterface/data/evt_2014.pdl'),
+            user_decay_embedded= cms.vstring(
+"""## my decay
+Alias      MyB+    B+
+Alias      MyB-   B-
+ChargeConj MyB-   MyB+
+#
+Alias      MyJpsi J/psi
+ChargeConj MyJpsi MyJpsi
+#
+# Alias      Myrho0       rho0
+# ChargeConj Myrho0       Myrho0
+#
+#
+Alias       Mypsi(2S)  psi(2S)
+ChargeConj  Mypsi(2S)  Mypsi(2S)
+#
+Decay MyJpsi
+1.000  mu+       mu-                     PHOTOS VLL;
+Enddecay
+#
+Decay MyB+
+1.000     Mypsi(2S) K+         PHSP;
+Enddecay
+#
+Decay MyB-
+1.000     Mypsi(2S) K-         PHSP;
+Enddecay
+#
+Decay Mypsi(2S)
+1.000   MyJpsi  pi+   pi-         VVPIPI;
+Enddecay
+#
+# Decay Myrho0
+# 1.000   pi+   pi-                  PHSP;
+# Enddecay
+#
+End"""
+     ),
+            list_forced_decays = cms.vstring('MyB+','MyB-'),
+            operates_on_particles = cms.vint32(),
+            convertPythiaCodes = cms.untracked.bool(False)
+            ),
+        parameterSets = cms.vstring('EvtGen130')
+        ),
+        PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            "SoftQCD:nonDiffractive = on",
+            'PTFilter:filter = on', # this turn on the filter
+            'PTFilter:quarkToFilter = 5', # PDG id of q quark
+            'PTFilter:scaleToFilter = 1.0'),
+        parameterSets = cms.vstring(
+            'pythia8CommonSettings',
+            'pythia8CP5Settings',
+            'pythia8PSweightsSettings',
```

Comment:
Hi @mtosi . Let me know if you prefer keeping them. If so, I am happy to sign. I think BPH doesn't use them for systematics.